x/lockup/keeper/lock_test.go

```diff
@@ -533,6 +533,60 @@ func (suite *KeeperTestSuite) TestCreateLock() {
 	suite.Require().Equal(sdk.NewInt(30), balance.Amount)
 }
 
+func (suite *KeeperTestSuite) TestCreateLockNoSend() {
```

Comment:
Is there a particular reason you made these non table driven?

Reply:
Yes, the pre-existing `TestCreateLock` was not table driven, and I wanted to use the exact logic flow to ensure this method behaves just like `CreateLock`, just without the send